view/frontend/templates/product/viewed.phtml

```diff
@@ -1,7 +1,12 @@
 <?php
 
-/** @var \Magento\Catalog\Helper\Image $_imagehelper */
-$_imagehelper = $this->helper('Magento\Catalog\Helper\Image');
+use Magento\Catalog\Helper\Image;
+use Magento\Catalog\Model\Product;
```

Comment:
do we need this as it should already be installed on the page? but if it's for being more explicit in readability, it's cool with me.

Reply:
we import the classes to be able to use the short notation in the code below